src/errors/runtime/message.js

```diff
@@ -31,5 +32,7 @@ export default {
     optionValueIsNotABoolean:            '"{name}" option is expected to be a boolean, but it was {type}.',
     optionValueIsNotANonNegativeNumber:  '"{name}" option is expected to be a non-negative number, but it was {type}.',
     optionValueIsNotAnObject:            '"{name}" option is expected to be an object, but it was {type}.',
-    optionValueIsNotAStringOrRegExp:     '"{name}" option is expected to be a string or regular expression, but it was {type}.'
+    optionValueIsNotAStringOrRegExp:     '"{name}" option is expected to be a string or regular expression, but it was {type}.',
+    browserOpeningIsNotSupportedOnLinux: 'The browser opening functionality are not yet supported on Linux. Subscribe to the following issue to keep track: https://github.com/DevExpress/testcafe-browser-natives/issues/12',
```

Comment:
We can't open local browsers yet: https://github.com/DevExpress/testcafe-browser-natives/blob/master/src/api/open.js#L47. But remote browsers are working.